src/main/resources/crafter/studio/studio-services-context.xml

```diff
@@ -280,6 +280,8 @@
     <bean id="studioEventService" class="org.craftercms.studio.impl.v1.service.event.EventServiceImpl" />
 
     <bean id="configurationService" class="org.craftercms.studio.impl.v2.service.configuration.ConfigurationServiceImpl">
+        <constructor-arg ref="crafter.configurationReader"/>
```

Comment:
Why mixing constructor args and properties? It should be either all constructor agrs or all properties

Reply:
I have been using constructor args for all new code because Spring is recommending it for required properties (they deprecated the `@Required` property that was used before) so in the future when we upgrade Spring it would be easier, I'm not sure if we want to refactor all existing code right now. @sumerjabri @avasquez614 any thoughts?